### PR TITLE
Sponsorship chris

### PR DESCRIPTION
**Pull request by chrispierrelouis26**
sponsorship

### CODE REVIEW

**Comment by lloan** [comment]
merged

File: pages/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // eslint-disable-next-line no-unused-vars
 import React, {useState, useEffect} from 'react';
 import Footer from '../src/components/footer';
```
Comment (lloan) [inline pages/index.js]: nit: spacing is wrong here - use ES

File: sass/components/sponsorship.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        }
+    }
+    #sponsor-title {
+        font-size: 26px;
+        font-family: "black";
+        color: white;
+        padding-top: 16px; 
+         
+    }
+    button {
```
Comment (lloan) [inline sass/components/sponsorship.scss]: test review comment

File: pages/index.js (hunk 1 above)
Comment (chrispierrelouis26) [reply pages/index.js]: ES?

**Comment by lloan** [comment]
@chrispierrelouis26 sorry - was showing another member how to work with reviews and used your PR as an example. Thanks!

**Pull request merged**